### PR TITLE
Slider accent color and ticks fix

### PR DESCRIPTION
**Pull request by dipeshmsft**
Fixes # 

Main PR

### CODE REVIEW

**Comment by harshit7962** [comment]
Due to these changes mouse over effect is discarded.

**Review by harshit7962 — CHANGES_REQUESTED** [review]

File: src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.FluentWindows/Styles/Slider.xaml [hunk 1]
```diff
@@ -24,28 +25,24 @@
...
-        <Setter Property="Foreground" Value="{DynamicResource SliderThumbBackground}" />
+        <Setter Property="Background" Value="{DynamicResource SliderThumbBackground}" />
         <Setter Property="BorderThickness" Value="1" />
-        <Setter Property="Background" Value="{DynamicResource SliderOuterThumbBackground}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="{x:Type Thumb}">
                     <Border
-                        Background="{TemplateBinding Background}"
+                        Background="{DynamicResource SliderOuterThumbBackground}"
```
[the remark this answers is not in the document]
Comment (dipeshmsft) [reply src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.FluentWindows/Styles/Slider.xaml:36]: Matching how WinUI does it.

**Comment by dipeshmsft** [comment]
I have added animation instead for feedback. Similar to how WinUI does it.

**Review by harshit7962 — APPROVED** [review]
Verified the changes in sample application, the visual feedback on pressed and mouse over is appropriate.

**Pull request merged**